vendor/github.com/rs/rest-layer-hystrix/hystrix.go

```diff
@@ -0,0 +1,148 @@
+// Package restrix is a REST Layer resource storage wrapper to add hystrix support
```

Comment:
I vendored those two small libs to make tests passes and avoid sync issues with those two projects. The rest-layer-mem is actually only useful in the context of unit tests and examples, so it should not be an issue. For hystrix, we might just move the example in the rest-layer-hystrix package instead, and unvendor it.